generators/server/index.js

```diff
@@ -342,6 +342,126 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
+                            dockerOthers.push(`npm run docker:${dockerConfig}`);
+                        }
+                    }
+                );
+                scriptsStorage.set({
+                    'predocker:others': dockerBuild.join(' && '),
+                    'docker:others': dockerOthers.join(' && '),
+                    'ci:e2e:prepare:docker': 'npm run docker:db && npm run docker:others && docker ps -a',
+                });
+            },
+            packageJsonBackendScripts() {
+                const packageJsonStorage = this.createStorage('package.json');
+                const scriptsStorage = packageJsonStorage.createStorage('scripts');
+                const javaCommonLog = `-Dlogging.level.ROOT=OFF -Dlogging.level.org.zalando=OFF -Dlogging.level.io.github.jhipster=OFF -Dlogging.level.${this.jhipsterConfig.packageName}=OFF`;
+                const javaTestLog =
+                    '-Dlogging.level.org.springframework=OFF -Dlogging.level.org.springframework.web=OFF -Dlogging.level.org.springframework.security=OFF';
+
+                const buildTool = this.jhipsterConfig.buildTool;
+                let e2ePackage = 'target/e2e';
+                if (buildTool === 'maven') {
+                    scriptsStorage.set({
+                        'backend:info': './mvnw -ntp enforcer:display-info --batch-mode',
+                        'backend:doc:test': './mvnw -ntp javadoc:javadoc --batch-mode',
+                        'backend:nohttp:test': './mvnw -ntp checkstyle:check --batch-mode',
+                        'java:jar': './mvnw -ntp verify -DskipTests --batch-mode',
+                        'java:war': './mvnw -ntp verify -DskipTests --batch-mode -Pwar',
+                        'java:docker': './mvnw -ntp verify -DskipTests jib:dockerBuild',
+                        'backend:unit:test': `./mvnw -ntp -P-webpack verify --batch-mode ${javaCommonLog} ${javaTestLog}`,
+                    });
+                } else if (buildTool === 'gradle') {
+                    const excludeWebpack = this.jhipsterConfig.skipClient ? '' : '-x webpack';
+                    e2ePackage = 'e2e';
+                    scriptsStorage.set({
+                        'backend:info': './gradlew -v',
+                        'backend:doc:test': `./gradlew javadoc ${excludeWebpack}`,
+                        'backend:nohttp:test': `./gradlew checkstyleNohttp ${excludeWebpack}`,
+                        'java:jar': './gradlew bootJar -x test',
+                        'java:war': './gradlew bootWar -Pwar -x test',
+                        'java:docker': './gradlew bootJar jibDockerBuild',
+                        'backend:unit:test': `./gradlew test integrationTest ${excludeWebpack} -Ptestcontainers ${javaCommonLog} ${javaTestLog}`,
+                        'postci:e2e:package': 'cp build/libs/*SNAPSHOT.$npm_package_config_packaging e2e.$npm_package_config_packaging',
```

Comment:
missing the same for maven ?

Reply:
No, maven I created a profile that creates e2e.jar directly.